### PR TITLE
Changed formatting in Alumni page

### PR DESCRIPTION
**Pull request by suldanaa**
Along with other changes such as relinking/removing links from footer and header respectively, and changing font in the header.

### CODE REVIEW

File: grossmont/computer-science-and-cybersecurity-organization/pages/alumni/alumni.css [hunk 1]
```diff
@@ -11,10 +11,10 @@
     flex-direction: row;
 }
 
-.image {
+img {
```
Comment (khiemgluong) [inline grossmont/computer-science-and-cybersecurity-organization/pages/alumni/alumni.css:14]: Very nice refactoring

File: grossmont/computer-science-and-cybersecurity-organization/pages/alumni/alumni.html [hunk 2]
```diff
@@ -64,34 +64,36 @@ <h3>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod temp
     <!--third member-->
     <div class="container" style="margin-top: 50px; margin-bottom: 50px;">
       <div>
-        <img class="image" ;
-          src="https://fiverr-res.cloudinary.com/images/q_auto,f_auto/gigs/173653543/original/9de3c4b3dad756c1d969fc469916f4a15bef899b/draw-bad-stick-figures-that-kinda-looks-like-you.png">
+        <img
+          src="https://media.discordapp.net/attachments/761470887001325569/1088329748943474768/IMG_2358.png?width=994&height=1092">
```
Comment (khiemgluong) [inline grossmont/computer-science-and-cybersecurity-organization/pages/alumni/alumni.html:68]: Looks really nice! I might need someone to shoot a full body photo of me if that's the photo theme we are going with

File: grossmont/computer-science-and-cybersecurity-organization/pages/alumni/alumni.html [hunk 3]
```diff
@@ -64,34 +64,36 @@ <h3>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod temp
...
       <div class="text">
         <h1>Suldana Afrah</h1>
-        <h2 style="color: #006A65;">Secretary</h2>
-        <h3>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
-          dolore magna aliqua. Tempor commodo ullamcorper a lacus vestibulum. Potenti nullam ac tortor vitae purus
-          faucibus ornare suspendisse. Dui vivamus arcu felis bibendum. Laoreet sit amet cursus sit amet dictum sit.
+        <h2>Secretary</h2>
+        <h3>
+          Hi there! I'm Suldana and I've been the CS Club secretary for 2 semesters, and am a 2nd year CS Student. I'm pursuing a BS in CompSci, 
+          mostly because I like programming <strike>and money</strike>. The club has offered me many connections and fun memories,
```
Comment (khiemgluong) [inline grossmont/computer-science-and-cybersecurity-organization/pages/alumni/alumni.html:77]: the money is the whole reason tbh

**Review by khiemgluong — APPROVED** [review]
I appreciate the refactoring, this helps alot with code readability and verbosity

**Comment by khiemgluong** [comment]
I'd say the overall look and feel of the club page is solid now. Maybe we can change the background of the body to white, but I think we should definitely start brainstorming a kind of blog page where your notes can be seen along with photos, also some ideas for what to put in the carousel in the home page.

File: grossmont/computer-science-and-cybersecurity-organization/pages/alumni/alumni.html (hunk 2 above)
Comment (suldanaa) [reply grossmont/computer-science-and-cybersecurity-organization/pages/alumni/alumni.html:68]: Doesn't have to be full body! I was hoping for something casual and not extremely formal, could be a selfie with something fun in the background or just you smiling and holding a thumbs up (though if you really wanted to, keeping your linkedin pfp would be ok!)

File: grossmont/computer-science-and-cybersecurity-organization/pages/alumni/alumni.html (hunk 3 above)
Comment (suldanaa) [reply grossmont/computer-science-and-cybersecurity-organization/pages/alumni/alumni.html:77]: > the money is the whole reason tbh

LOL

**Comment by suldanaa** [comment]
> I'd say the overall look and feel of the club page is solid now. Maybe we can change the background of the body to white, but I think we should definitely start brainstorming a kind of blog page where your notes can be seen along with photos, also some ideas for what to put in the carousel in the home page.

I was actually speaking to a friend of mine who majored in digital media with a focus on web design! she works as a ui/ux designer now. she made a lot of suggestions, specifically about the color of the website, some of the font choices, etc. One of the suggestions was that our website was very low contrast (blue on blue) and it caused issues with accessibility, so a white background would be a good start. I was going to propose these ideas to you all since they're somewhat big changes in terms of the site's color scheme and visual!

**Pull request merged**